go.mod

```diff
@@ -129,10 +129,10 @@ require (
 	golang.org/x/text v0.14.0
 	golang.org/x/time v0.5.0
 	golang.org/x/tools v0.19.0
-	golang.stackrox.io/grpc-http1 v0.3.9
+	golang.stackrox.io/grpc-http1 v0.3.10
 	google.golang.org/api v0.157.0
-	google.golang.org/genproto v0.0.0-20240116215550-a9fa1716bcac
-	google.golang.org/grpc v1.61.1
+	google.golang.org/genproto v0.0.0-20240123012728-ef4313101c80
+	google.golang.org/grpc v1.62.1
```

Comment:
I think this is a problem we need to first update protobuf 
cc @RTann

Reply:
I don't follow. Why do we need to update protobuf first?